src/seismometer/controls/explore.py

```diff
@@ -1425,3 +1425,175 @@ def generate_plot_args(self) -> tuple[tuple, dict]:
 
 
 # endregion
+
+# region Binary Model Metric Exploration Widget
+
+
+class BinaryModelMetricOptions(Box, ValueWidget):
+    value = traitlets.Dict(help="The selected values for the audit and model options")
+
+    def __init__(
+        self,
+        cohort_groups: dict[str, tuple[Any]],
+        target_names: tuple[Any],
+        score_names: tuple[Any],
+        per_context: bool = True,
+        metrics: tuple[str] = None,
+        default_metrics: tuple[str] = None,
+    ):
+        """
+        Widget for selecting interventions and outcomes across categories in a cohort group.
+
+        Parameters
+        ----------
+        target_names : tuple[Any]
+            model target columns
+        score_names : tuple[Any]
+            model score columns
+        score_threshold : float
+            main threshold for the model score
+        per_context : bool, optional
+            if scores should be grouped by context, by default True
+        metrics : tuple[str], optional
+            list of fairness metrics to display, if None (default) metrics from `calculate_bin_stats`
+        default_metrics : tuple[str], optional
+            list of fairness metrics to display, if None (default) will use ["tpr", "fpr", "pprev"]
+        """
+        metrics = metrics or [
```

Comment:
Consider updating to use the MetricGenerator from https://github.com/epic-open-source/seismometer/pull/99/files#diff-5c6deaba7b56c0d09f61cb09459e3a588d4196ff2c07d088b0db06377d48b6ebR94